src/os/ui/ogc/iogcdescriptor.js

```diff
@@ -234,6 +234,7 @@ os.ui.ogc.IOGCDescriptor.prototype.getLayerName;
 
 /**
  * @return {?string}
+ * @deprecated Use IFilterable::getFilterKey()
```

Comment:
We don't have anything else that references methods in this manner. For consistency, I'd prefer something like `IFilterable.getFilterKey()` instead.

Reply:
Updated.